dev-db/sqlitebrowser/sqlitebrowser-3.9.1.recipe

```diff
@@ -0,0 +1,63 @@
+SUMMARY="DB Browser for SQLite"
+DESCRIPTION="DB Browser for SQLite is a high quality, visual, open source tool \
+to create, design, and edit database files compatible with SQLite."
+HOMEPAGE="http://sqlitebrowser.org/"
+COPYRIGHT="2012-2016 Martin Kleusberg"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/sqlitebrowser/sqlitebrowser/archive/v3.9.1.tar.gz"
+CHECKSUM_SHA256="d0d2e06a69927ba1d0b955f3261ce70c61befc5bd5ddaa06752dae8bb4219ed8"
+PATCHES="sqlitebrowser-3.9.1.patchset"
+ADDITIONAL_FILES="sqlitebrowser.rdef"
```

Comment:
Shouldn't I also change the name of `sqlitebrowser.rdef` when `rc` is called in `BUILD()`?